tests/imodels-clients-tests/src/integration/management/IModelOperations.test.ts

```diff
@@ -520,7 +520,7 @@ describe("[Management] IModelOperations", () => {
       }
     });
 
-    const newIModelName = testIModelGroup.getPrefixedUniqueIModelName("new iModel name for search");
+    const newIModelName = testIModelGroup.getPrefixedUniqueIModelName("TEST iModel for update and for search Updated");
```

Comment:
this name is also a bit hard to read at this point

Reply:
updated